### PR TITLE
Updated flowers and info command in UserGuide.md

### PR DESCRIPTION
**Pull request by samuelory**

### CODE REVIEW

**Review by IanFH — APPROVED** [review]
UG looks good to me

**Review by JeffinsonDarmawan — APPROVED** [review]
Good addition of userguide for flowers

File: docs/UserGuide.md [hunk 1]
```diff
@@ -69,31 +69,57 @@ Here are the list of your saved bouquets:
...
+1. White Orchid
+2. Dark crimson Rose
+3. Red Rose
+4. White Lily
+5. White Daisy
+Type 'next' to go to the next page.
+____________________________________________________________
+```
+
+Example: `flowers Funeral`
```
Comment (Ijaaz01) [inline docs/UserGuide.md:93]: Very detailed, good job!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -69,31 +69,57 @@ Here are the list of your saved bouquets:
...
+3. White Chrysanthemum
+____________________________________________________________
 ```
 ### View detailed info of a flower: info
 
-Get detailed info (colour, meaning etc) about a specific flower in the database
+Get detailed info (colour, meaning and related occasions) about a specific flower in the database,
+will show all colours and their associated meanings (specifying colours will come soon).
+
+Shows at most five flowers at once, type `next` to see flowers on the next page, and `back` to view a previous page.
```
Comment (Ijaaz01) [inline docs/UserGuide.md:108]: Clear instructions well done!

**Review by Ijaaz01 — APPROVED** [review]

**Pull request merged**